.circleci/images/runner/Dockerfile

```diff
@@ -20,9 +20,6 @@ RUN chmod +x /bin/gimme
 RUN gimme $GIMME_GO_VERSION
 RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
 
-# Download and install golangci-lint
-RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $GOPATH/bin v1.21.0
-
```

Comment:
You need to manually build and push the CircleCI runner and update it on the config following the [README](https://github.com/DataDog/datadog-agent/tree/master/.circleci#upgrading-golang-version), since this image is not built automatically